parts/k8s/kubernetesinstalls.sh

```diff
@@ -278,6 +278,9 @@ installKubeletAndKubectl() {
 pullContainerImage() {
     CLI_TOOL=$1
     DOCKER_IMAGE_URL=$2
+    if [ ! -z "${PRIVATE_AZURE_REGISTRY_SERVER}" ]; then
+        $CLI_TOOL login -u $SERVICE_PRINCIPAL_CLIENT_ID -p $SERVICE_PRINCIPAL_CLIENT_SECRET $PRIVATE_AZURE_REGISTRY_SERVER
```

Comment:
should this use `--password-stdin` here?  https://docs.docker.com/engine/reference/commandline/login/#provide-a-password-using-stdin